File: beecrowd/1/1013/1013.c
#include <stdio.h>
#include <stdlib.h>

int maiorAB(int A, int B) {
    return ((A + B + abs(A - B))) / 2;
}

int main(void) {
    int num[3];
    int maior;
    scanf("%d %d %d%*c",&num[0], &num[1], &num[2]);
    maior = maiorAB(num[0],num[1]);
    maior = maiorAB(maior,num[2]);
    printf("%d eh o maior\n", maior);
    return 0;
}

File: beecrowd/1/1013/1013.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

class Main {

    public static int maiorAB(int A, int B) {
        return ((A + B + Math.abs(A - B))) / 2;
    }

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);
        int num[] = new int[3];
        int maior;
        for (int i = 0; i < 3; i++) {
            num[i] = sc.nextInt();
        }

        maior = maiorAB(num[0], num[1]);
        maior = maiorAB(maior, num[2]);

        System.out.printf("%d eh o maior\n", maior);
        sc.close();

    }

}
